.github/workflows/e2e_network_tests.yml

```diff
@@ -30,7 +30,7 @@ jobs:
         INFURA_SEPOLIA_HTTP: ${{ secrets.INFURA_SEPOLIA_HTTP }}
         INFURA_MAINNET_HTTP: ${{ secrets.INFURA_MAINNET_HTTP }}
         MODE: ${{ matrix.mode }}
-        TEST_ACCOUNT_ADDRESS: ${{ fromJSON('{"sepolia":"0xa127C5E6a7E3600Ac34A9a9928E52521677e7211","mainnet":"0x98AF911164f9d4E0f5983ed114949c3Bfe3ADc9d"}')[matrix.network] }}
+        TEST_ACCOUNT_ADDRESS: ${{ fromJSON('{"sepolia":"0x0000000000000000000000000000000000000000","mainnet":"0x0000000000000000000000000000000000000000"}')[matrix.network] }}
```

Comment:
Blackbox Tests were not failing on sepolia, could you double check old logs for it. and run tests with above change. thanks

Reply:
Thanks @jdevcs , yes tests where not failing for sepolia. But I thought to use an address that would never cause the tests to fail in the future as a precaution. Or what do you think?
And yes, I run the tests and they all run with success here: https://github.com/web3/web3.js/actions/runs/7419127171/job/20188191757?pr=6689
Thanks,